spec/support/pageflow/dummy/rails_template.rb

```diff
@@ -8,17 +8,23 @@ def source_paths
 
 # Setup database.yml to use credentials from env variable.
 
-gsub_file('config/database.yml',
-          /^  database: /,
-          "  database: #{ENV.fetch('PAGEFLOW_PLUGIN_ENGINE', 'pageflow')}-")
+copy_file('database.yml', 'config/database.yml', force: true)
 
-gsub_file('config/database.yml',
-          /^  username:.*\n/,
-          "  username: \"<%= ENV.fetch('PAGEFLOW_DB_USER', 'root') %>\"\n")
+rails_version_string = Rails::VERSION::STRING
+rails_version_string.tr!('.', '_')
```

Comment:
Let's put the `tr` at the end of the previous line. Then we also don't need the bang version.